src/patternfly/components/Drawer/drawer.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-drawer {
+  --pf-c-drawer__main--MarginRight: calc(var(--pf-c-drawer__panel--Width) * -1);
+  --pf-c-drawer--m-open__main--MarginRight: 0;
+  --pf-c-drawer__main--Transition: margin .25s ease;
+  --pf-c-drawer__panel--ZIndex: var(--pf-global--ZIndex--sm);
+  --pf-c-drawer__panel--Width: 100%;
+  --pf-c-drawer__panel--md--Width: #{pf-size-prem(450px)};
+  --pf-c-drawer__panel--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-drawer__panel--Transition: transform .25s ease;
+  --pf-c-drawer__panel--Transform: translateX(100%);
+  --pf-c-drawer--m-open__panel--Transform: translateX(0);
```

Comment:
should all the `--pf-c-drawer--m-open..` go before anything else in this list

Reply:
It can, I don't feel strongly about it. I grouped it by the value the var modifies, not the selector, so that the modifier var is after the thing it modifies. Same with

```
  --pf-c-drawer__main--MarginRight: calc(var(--pf-c-drawer__panel--Width) * -1);
  --pf-c-drawer--m-expanded__main--MarginRight: 0;
```

Want me to re-arrange them?

Reply:
All good